program_intent_engagement/apps/api/v1/tests/test_views.py

```diff
@@ -0,0 +1,155 @@
+"""
+Tests for the Program-Intent API views
+"""
+import datetime
+import uuid
+
+import ddt
+from django.urls import reverse
+
+from program_intent_engagement.apps.core.models import ProgramIntent
+from test_utils import ProgramIntentAPITestCase
+from test_utils.factories import UserFactory
+
+
+@ddt.ddt
+class ProgramIntentRecentAndCertainViewTests(ProgramIntentAPITestCase):
+    """
+    Tests MostRecentAndCertainIntentsView
+    """
+
+    def setUp(self):
+        super().setUp()
+
+        self.program_id1 = str(uuid.uuid4())
+        self.program_id2 = str(uuid.uuid4())
+
+        self.p1_intent_maybe = ProgramIntent.objects.create(
+            lms_user_id=self.user.lms_user_id,
+            program_uuid=self.program_id1,
+            reason="a reason",
+            certainty="MAYBE",
+            effective_timestamp=datetime.datetime(2020, 8, 17),
+        )
+
+    def get_api(self, user):
+        """
+        Helper function to make a get request to the API
+        """
+        url = reverse("api:v1:intents-list")
+
+        headers = self.build_jwt_headers(user)
+        return self.client.get(url, **headers)
+
+    def get_response(self, user, expected_response):
+        """
+        Helper function to get API response
+        """
+
+        response = self.get_api(user)
+        self.assertEqual(response.status_code, expected_response)
+
+        return response
+
+    def test_user_no_intents(self):
+        """
+        Test that when user has no program intents you get 200 ok response with an empty list
+        """
+        new_user = UserFactory()
+
+        expected_response = []
+        actual_response = self.get_response(new_user, 200)
+
+        self.assertEqual(len(actual_response.data), 0)
+        self.assertEqual(actual_response.data, expected_response)
```

Comment:
This line is implicitly testing the same thing as the above line, which makes the above line unnecessary. This line asserts that the actual response data is equal to the empty list. If that is true, then the actual response data will always have a length of 0, because it's the empty list. I think this one line sufficiently tests what you want to test: that there is no data returned by the API.